deepmd/utils/argcheck.py

```diff
@@ -104,6 +105,13 @@ def type_embedding_args():
         Argument(
             "use_econf_tebd", bool, optional=True, default=False, doc=doc_use_econf_tebd
         ),
+        Argument(
+            "use_tebd_bias",
+            bool,
+            optional=True,
+            default=False,
+            doc=doc_use_tebd_bias,
+        ),
```

Comment:
While the default value of `activation_function` is `tanh` and the default value of `use_tebd_bias` is `False`, isn't the output value limited to $(-1,1)$?

Reply:
It should not be a big issue to eliminate the intermediate type representation (embedded types)  in range (-1, 1). 
In the subsequent usages, e.g. in embedding nets, the intermediate type rep. firstly passes linear operator in the first layer of the embedding net. Therefore, we do not need to add a linear layer after the activation.

Reply:
In fact, we use `none` as default value of `activation_function` when user **does not set the `type_embedding` key** in TF, see https://github.com/deepmodeling/deepmd-kit/blob/devel/deepmd/tf/model/model.py#L696 .

Reply:
This is a strange behavior. I do no understand why the activation function is set to None if user does not set the type_embedding